dotnet/src/Azure.Iot.Operations.Services/AssetAndDeviceRegistry/AdrServiceClient.cs

```diff
@@ -268,7 +270,7 @@ public async Task<Asset> UpdateAssetStatusAsync(string deviceName, string inboun
     }
 
     /// <inheritdoc />
-    public async Task<CreateDetectedAssetResponse> CreateDetectedAssetAsync(string deviceName, string inboundEndpointName, CreateDetectedAssetRequest request,
+    public async Task<CreateDetectedAssetResponse> CreateOrUpdateDiscoveredAssetAsync(string deviceName, string inboundEndpointName, CreateOrUpdateDiscoveredAssetRequest request,
```

Comment:
Since this method is inheriting its doc from IAdrServiceClient, then these same changes should be in IAdrServiceClient as well.